content/spire/try/oidc_federation_aws.md

```diff
@@ -0,0 +1,399 @@
+---
+title: SPIRE OIDC Federation with AWS
+short: Federation
+description: Setting up SPIRE to federate with AWS
+weight: 50
+toc: true
+menu:
+  spire:
+    weight: 20
+    parent: 'spire-try'
+---
+Since version 0.8.1, SPIRE contains experimental support for [Open ID Connect](https://openid.net/connect/) (OIDC) federation and SPIRE Server-to-SPIRE Server federation. This document describes how to authenticate SPIRE workloads to Amazon AWS S3 buckets using OIDC so the workloads can use data on S3. Using this document as a guide, you should be able to configure access to AWS services other than S3.
+
+We assume the instances running workloads are on a cloud service other than Amazon, such as GCP. If the instances were on Amazon, you could use IAM to authenticate workloads rather than SPIFFE OIDC federation.
+
+As OIDC is based on OAuth, SPIRE workloads use JWT [SVIDs](https://spiffe.io/spiffe/#spiffe-verifiable-identity-document-svid) rather than X.509 SVIDs to transmit workload identity information.
+
+For more information about configuring SPIRE OIDC federation, watch a 22-minute [demo of OIDC authentication](https://www.youtube.com/watch?v=db_3LefoG9k&list=PLWsNZXV-gXVY_br7I8gz9q0Fijk4DoUxG&index=9&t=775s) to AWS S3 buckets.
+
+## Related Documents
+
+As part of setting up SPIRE OIDC federation, you must install and configure the SPIRE OIDC [Discovery Provider REST API](/spire/docs/oidc_discovery/).
+
+Complete installation and configuration of the SPIRE Server and Agent are not described in this document. See the [SPIRE documentation](/spire/docs/) for installation and configuration information.
```

Comment:
Though we certainly don't want to repeat ourselves unnecessarily, it's important that a tutorial provide self-contained specific steps for new users. I'd suggest that we solve for this by asking the user first to go through one of the earlier Quickstarts (ideally the Kubernetes one) and to start this tutorial at the end of that. That we can assume a bunch of the basics of their environment have already been set up, and we need only here to show how to modify/extend that environment to solve for the goal of the tutorial.